openaz-xacml/src/main/java/org/apache/openaz/xacml/std/jaxp/JaxpRequest.java

```diff
@@ -114,19 +123,13 @@ public static JaxpRequest load(File fileXmlRequest) throws ParserConfigurationEx
             logger.error("No Document returned parsing \"" + fileXmlRequest.getAbsolutePath() + "\"");
             return null;
         }
-
-        NodeList nodeListRoot = document.getChildNodes();
-        if (nodeListRoot == null || nodeListRoot.getLength() == 0) {
-            logger.warn("No child elements of the XML document");
-            return null;
+        
+        Node nodeRoot = document.getDocumentElement();
+        
+        if(context == null) {
+            init();
```

Comment:
Why adding separate method, just call in `context = JAXBContext.newInstance(RequestType.class);`  here.